verification/src/transaction_verifier.rs

```diff
@@ -103,20 +108,43 @@ impl<'a> VersionVerifier<'a> {
     }
 }
 
+pub struct SizeVerifier<'a> {
+    transaction: &'a Transaction,
+    block_bytes_limit: u64,
+}
+
+impl<'a> SizeVerifier<'a> {
+    pub fn new(transaction: &'a Transaction, block_bytes_limit: u64) -> Self {
+        SizeVerifier {
+            transaction,
+            block_bytes_limit,
```

Comment:
If we use consensus `max_block_bytes` here, the transaction which size approaching bytes limit will be accepted and broadcasted to peers, but cannot be assembled in `BlockTemplate` because of the extra cellbase size.

How about adding cellbase size limit to consensus and minus this value here as gap?

Reply:
Let's address this in https://github.com/nervosnetwork/ckb/issues/885